binderhub/log.py

```diff
@@ -78,6 +78,8 @@ def log_request(handler):
     """
     status = handler.get_status()
     request = handler.request
+    request_time = 1000.0 * handler.request.request_time()  # milliseconds
```

Comment:
```suggestion
    request_time = 1000.0 * handler.request.request_time()  # micro- to milli-seconds
```

Is this correct?

Reply:
No, seconds to milliseconds. Added clarifying comment

Reply:
Woops ah